subsystems/sound/etc/containers/systemd/audio.container

```diff
@@ -4,6 +4,7 @@ After=network.target
 
 [Container]
 Image=quay.io/qm-images/audio:latest
+Network=host
 
```

Comment:
@ArtiomDivak
Shouldn't this quadlet contain `AddDevice=/dev/snd` ?
I was running manually the sound container with this quadlet:
`# sound-test.container
[Container]
AddDevice=/dev/snd
ContainerName=sound-test
Image=localhost/local-audio-image
`

Reply:
You are right

Reply:
https://github.com/containers/qm/pull/787
Added this MR to fix it